extensions/sql-migration/src/constants/strings.ts

```diff
@@ -832,6 +832,14 @@ export function SQL_MIGRATION_SERVICE_DETAILS_HEADER(sqlMigrationServiceName: st
 	return localize('sql.migration.service.header', "Azure Database Migration Service \"{0}\" details:`", sqlMigrationServiceName);
 }
 export const DATABASE_MIGRATION_SERVICE_AUTHENTICATION_KEYS = localize('sql.migration.database.migration.service.authentication.keys', "Database Migration Service authentication keys");
+
+// configure IR dialog
+export const POWERSHELL_SCRIPT_DESCRIPTION = localize('sql.ir.powershellscript.definition', "Once executed in powershell, the unique script shown below will download, and install the Self-hosted integration Runtime and register it with the Azure Database Migration Service. You will need to have PowerShell installed on the target machine.");
```

Comment:
should adjust case of "self-hosted integration runtime" to all lower.

Reply:
Same for other references below.

Reply:
Done